### ISSUE TITLE
Graph Type Naming is Very Manual (i.e. not very ottomatic)

### ISSUE THREAD

**Issue by floyd-may** [asemio/otto-the-geek#13]
Given some folder structure containing model classes like:

```
FeatureA
  - Thing.cs
  - OtherThing.cs
FeatureB
  - Thing.cs
```

It's very easy to confuse otto by including both `FeatureA.Thing` and `FeatureB.Thing` into your model. Both will get GraphQL graph type names of `Thing`. This is probably bad. Two things really need to happen here:

- [x] Make Otto throw a validation error when this happens
- [ ] Make it easy to define a naming convention to circumnavigate this problem, maybe by namespace prefixing?

**Comment by floyd-may** [comment]
addressed in 5480729.

**Comment by floyd-may** [comment]
Closing - not convinced that naming conventions are the right thing.
